Manatee.Json/Patch/JsonPatchAction.cs

```diff
@@ -84,8 +84,20 @@ private JsonPatchResult _Add(JsonValue json)
 
 		private JsonPatchResult _Remove(JsonValue json)
 		{
-			var (target, key, index, found) = JsonPointerFunctions.ResolvePointer(json, Path);
-			if (!found) return new JsonPatchResult(json, $"Path '{Path}' not found.");
+            if (Path == "")
```

Comment:
Please fix indentation.  Tabs, not spaces.  If you have Resharper or use Rider, the settings should load with the solution.